src/gulp/spec.js

```diff
@@ -68,6 +68,10 @@ export default function(options) {
       cliOptions = { config: config }
     }
 
+    if (argv.runInBand) {
+      cliOptions.runInBand = true;
```

Comment:
if they don't run this with `--build` you will get a js error that cliOptions is not defined